pip/pip-342 OTel client metrics support.md

```diff
@@ -0,0 +1,167 @@
+| `pulsar.client.producer.opened`                 | Counter       | sessions    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`]                                                                            | The number of producer sessions opened                                                                                                    |
+| `pulsar.client.producer.closed`                 | Counter       | sessions    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`]                                                                            | The number of producer sessions closed                                                                                                    |
+| `pulsar.client.consumer.opened`                 | Counter       | sessions    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of consumer sessions opened                                                                                                    |
+| `pulsar.client.consumer.closed`                 | Counter       | sessions    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of consumer sessions closed                                                                                                    |
+| `pulsar.client.consumer.message.received.count` | Counter       | messages    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of messages explicitly received by the consumer application                                                                    |
+| `pulsar.client.consumer.message.received.size`  | Counter       | bytes       | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of bytes explicitly received by the consumer application                                                                       |
+| `pulsar.client.consumer.receive_queue.count`    | UpDownCounter | messages    | `pulsar.tenant`, `pulsar.namespace`, [`pulsar.topic`], [`pulsar.partition`], `pulsar.subscription`                                                     | The number of messages currently sitting in the consumer receive queue                                                                    |
```

Comment:
`_` should be `.`?

Reply:
No since it is to convery one word, and not build an hierarchy.